### PR TITLE
OKTA-79132 Add cleaned up error codes back in the OIDC doc

### PR DESCRIPTION
**Pull request by mohitchugh-okta**
Cleaned up the code as per OKTA-79132.

Made changes to both oidc and error_codes.md

Resolves: OKTA-79132

Primary reviewer: @joelfranusic-okta 
Reviewers: @tthompson-okta 

FYI - @mystiberry-okta

### CODE REVIEW

File: _source/docs/api/resources/oidc.md [hunk 1]
```diff
@@ -187,13 +187,15 @@ unsupported_response_mode  | The specified response mode is invalid or unsupport
 invalid_scope  | The scopes list contains an invalid or unsupported value.    | 
 server_error  | The server encountered an internal error.    | 
 temporarily_unavailable  | The server is temporarily unavailable, but should be able to process the request at a later time.    |
+invalid_request | The request is missing a necessary parameter or the parameter has an invalid value. |
+invalid_client  | The specified client id is invalid.
+access_denied   | The server denied the request.
```
Comment (tthompson-okta) [inline _source/docs/api/resources/oidc.md]: What about invalid_grant? I thought you mentioned we were currently returning that in some places.

**Comment by tthompson-okta** [comment]
Ah, I actually intended to update this page: http://developer.okta.com/docs/api/getting_started/error_codes.html#openid-connect-and-okta-social-authentication

I think the file is called `_source/docs/api/getting_started/error_codes.md`

File: _source/docs/api/resources/oidc.md (hunk 1 above)
Comment (mohitchugh-okta) [inline _source/docs/api/resources/oidc.md]: we only return it for the token endpoint

**Comment by mohitchugh-okta** [comment]
I can update that page as well, but that's a lot of duplication. Isn't it better to have the error-codes next to the API that is calling it rather than provide a list of out of context?

@karlmcguinness-okta FYI as well.

**Comment by mystiberry-okta** [comment]
The customer may receive an error and not know which object/operation it's associated with, yes?


Mysti


________________________________

**Comment by mohitchugh-okta** [comment]
I'm not sure. Karl and Joel would know more about the usage pattern of the SDK. I'm just worried that these two lists will inevitably go out of sync and end up causing more confusion. 

@joelfranusic-okta Is there a way to pull in the error codes in the two places from the same underlying master list?

**Comment by mystiberry-okta** [comment]
or we can link to one resource from the other location. I've been told there's no reuse in kramdown/markdown, but would love to be wrong!


Mysti


________________________________

**Comment by joelfranusic-okta** [comment]
I can think of at least two ways to reuse content in Jekyll. However I'm reluctant to fix it right now since a project that I'll be starting soon might make that all moot.

**Comment by mohitchugh-okta** [comment]
Yup, that's why I've just gone ahead and duplicated the codes for now. We'll deal with de-duplication later when we (hopefully) generate these codes directly from code.

File: _source/docs/api/resources/oidc.md (hunk 1 above)
Comment (tthompson-okta) [inline _source/docs/api/resources/oidc.md]: Ah right, I see it below in the token endpoint section.

File: _source/docs/api/getting_started/error_codes.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ This document provides further information about the errors that the Okta API re
 
 In stituations where Okta needs to pass an error to a downstream application via a `redirect_uri`, the error code and description will be encoded as the query parameters `error` and `error_description`.
 
-> For example, if the **redirect_uri** is **https://example.com**, then the **UNKNOWN_ENDPOINT** error would be passed as follows:
+> For example, if the **redirect_uri** is **https://example.com**, then the **ACCESS_DENIED** error would be passed as follows:
 > 
 > ~~~
 > https://example.com?error=UNKNOWN_ENDPOINT&error_description=Unknown%20social%20endpoint.
```
Comment (tthompson-okta) [inline _source/docs/api/getting_started/error_codes.md]: You need to update the example here to use ACCESS_DENIED.

Comment (mohitchugh-okta) [inline _source/docs/api/getting_started/error_codes.md]: Yup. Fixed.

**Comment by tthompson-okta** [comment]
LGTM. :sheep:

**Comment by mohitchugh-okta** [comment]
@mystiberry-okta Can you please merge this PR in? Thanks.

**Comment by mystiberry-okta** [comment]
Yes--can I rebuild site today or should I wait for Preview or Production?

**Comment by mystiberry-okta** [comment]
@eklein-okta will publish the site on Thursday.

**Pull request merged**